croo/croo.py

```diff
@@ -8,16 +8,20 @@
 
 import os
 import sys
+import logging
 import json
 import re
 import caper
-from caper.caper_uri import init_caper_uri, CaperURI, URI_LOCAL
-
+from autouri import AutoURI, AbsPath, GCSURI, S3URI, logger
 from .croo_args import parse_croo_arguments
 from .croo_html_report import CrooHtmlReport
 from .cromwell_metadata import CromwellMetadata
 
 
+logging.basicConfig(level=logging.INFO, format='%(asctime)s|%(name)s|%(levelname)s| %(message)s')
+logger = logging.getLogger('croo')
```

Comment:
Usually it is best to call `getLogger` with `__name__`, see https://docs.python.org/3/library/logging.html#logger-objects

```suggestion
logger = logging.getLogger(__name__)
```